pkg/api/v1alpha1/cluster_defaults.go

```diff
@@ -43,11 +42,3 @@ func setRegistryMirrorConfigDefaults(clusterConfig *Cluster) error {
 	}
 	return nil
 }
-
-func setWorkerNodeGroupDefaults(cluster *Cluster) error {
```

Comment:
let's keep this in, because this is the part that actually sets the default, in case the config is not created using generate cluster config command. Since we're removing the same logic from [vsphere validator](https://github.com/aws/eks-anywhere/pull/1021/files#diff-238036d622b0e49427d5c5115ba7fa29145e3741beb69f67aa345a6094280deeL91) it won't print the same msg twice